Packs/EWS/Integrations/EWSv2/README.md

```diff
@@ -390,6 +390,7 @@
 <h3 id="h_7bdec9fe-e3d9-4645-8da4-337ee3798a84">3. Get a list of searchable mailboxes</h3>
 <hr>
 <p>Returns a list of searchable mailboxes.</p>
+<p>Note. you should avoid running this command if you have over 1M mailboxes.</p>
```

Comment:
Any suitable replacement?

Reply:
no.
you can try with a big timeout but it’s strongly not recommended.